pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -44,7 +44,7 @@ python3.pkgs.buildPythonApplication rec {
       --replace "Flask~=1.1.2" "Flask~=2.0" \
       --replace "dateparser~=0.7" "dateparser>=0.7" \
       --replace "docker~=4.2.0" "docker>=4.2.0" \
-      --replace "requests==" "requests~=" \
+      --replace "requests==" "requests #" \
```

Comment:
Why not use >=

Reply:
This is more similar to the one below it (`==` -> ` #`, than `~=` -> `>=` ). No particular reason other than that.

Reply:
This also removes potential conditions after the requirement. I would just stick to >=.